src/views/partials/analytics.js

```diff
@@ -0,0 +1,39 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import AppConstants from '../../app-constants.js'
+
+export const analytics = `
```

Comment:
OK I just moved analytics to a partial so there isn't the duplication between guest/main, it means we'll listen for CTAs on the non-guest pages, but maybe we want to do that anyway?